### ISSUE TITLE
SwaggerResponse do not displays response model in case of scalar types.

### ISSUE THREAD

**Issue by LbISS** [domaindrivendev/Swashbuckle#815]
When i'm using SwaggerResponse attribute with any custom object  - it's working fine, but if i'm trying to return any of scalar types - response model and description are empty.

```
/// <summary>
/// Registers the specified user.
/// </summary>
/// <param name="model">The model.</param>
/// <returns></returns>
[AllowAnonymous]
[ActionName("Register")]
[SwaggerResponse(HttpStatusCode.BadRequest, "Error code", typeof(Int32))]
[SwaggerResponse(HttpStatusCode.OK, "User_ID", typeof(Int32))]
public async Task<IHttpActionResult> Register(RegisterBindingModel model)
{
	var user = new User { UserName = model.PhoneNumber, PhoneNumber = model.PhoneNumber };

	IdentityResult result = await _userManager.CreateAsync(user);

	if (!result.Succeeded)
		return GetErrorResult(result);
			
	return Ok(123);
}
```

With custom object:
![001](https://cloud.githubusercontent.com/assets/467427/16715265/6314c4e8-46e3-11e6-8108-23423e4bd8cc.PNG)

With scalar:
![002](https://cloud.githubusercontent.com/assets/467427/16715264/6314574c-46e3-11e6-8640-2da8e832e929.PNG)

**Comment by Demarsch** [comment]
+1
I could add that it also doesn't work for array of primitive types. At least it is not shown for byte[]

**Comment by Demarsch** [comment]
For some reason it shows up correct if you add breakpoint inside custom SchemaFilter
![image](https://cloud.githubusercontent.com/assets/4508264/16749182/bc5f6af4-47c8-11e6-8258-32717652c5be.png)
Even JSON schema generated shows this description

    "/api/v1/Files/{id}":{  
         "get":{  
            "tags":[  
               "Files"
            ],
            "summary":"Downloads file with specified Id and owned by current user",
            "operationId":"Files_Get",
            "consumes":[  

            ],
            "produces":[  
               "application/json",
               "text/json",
               "application/xml",
               "text/xml"
            ],
            "parameters":[  
               {  
                  "name":"id",
                  "in":"path",
                  "description":"Id of requested file",
                  "required":true,
                  "type":"integer",
                  "format":"int32"
               }
            ],
            "responses":{  
               "200":{  
                  "description":"Contents of specified file as byte array",
                  "schema":{  
                     "format":"byte",
                     "type":"string"
                  }
               },
               "403":{  
                  "description":"Unauthorized request"
               },
               "404":{  
                  "description":"File with specified Id doesn't exist"
               }
            },
            "deprecated":false
         }

But it doesn't show up on the web page
![image](https://cloud.githubusercontent.com/assets/4508264/16749240/177393c0-47c9-11e6-86af-825670843207.png)

**Comment by domaindrivendev** [comment]
This was a swagger-ui issue which has now been fixed. The version used by SB has now been upgraded to v2.2.0 so this version should be resolved as of SB 5.3.0
